docs/source/distributed_systems.rst

```diff
@@ -77,22 +76,23 @@ Proxies and multiple scripts
 ============================
 
 With distributed systems, it is very common to have multiple scripts (many
-times spread accross multiple machines). If those are to interact together,
+times spread across multiple machines). If those are to interact together,
 you should consider whether a single/shared name server should be used. This
 simplifies the way you can get proxies to every agent and also the way they
 may share information between them.
 
 If you are just starting with distributed systems and osBrain, you may
-approach it in the following way:
+approach it the following way:
 
 - You create one script, in which you run some agents and configure them.
-- You create another script, in which you run some agents and configure them.
+- You create another script, in which you run some more agents and configure
+  those as well.
 - From the second script, you access agents that you created on the first
-  and except that they are all ready and completely configured. But it turns
+  and expect them to be ready and completely configured. But it turns
   out that might not be the case.
 
-There is a very simple way to avoid this issue. Do create the agents from
-different scripts, but try to do all the configuration just from one of them.
+There is a very simple way to avoid this issue. You may create the agents from
+different scripts, but try to do all the configuration from one of them.
 Remember that proxies allow you to treat remote agents just like local objects,
-which means you do not really mind if they are running in one machine or
+which means it doesn't really matter if they are running in one machine or
```

Comment:
We try to avoid contractions in the documentation. So I would rather write "does not really matter". :wink:

Reply:
I'll keep that in mind.
Still, I think "does not really matter" sounds better than "you do not really mind"

Reply:
Sure! Go ahead with the change. Just avoid the contraction. :blush: